src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea(): number;
+}
 
+export function roundNumber(number : number) :number {
```

Comment:
Ехал Грека через реку,
Видит Грека – в реке number.            
Сунул Грека number в number, -
number number number number

rename your parameter, please)
```suggestion
export function roundNumber(number: number): number {
```